pkg/minikube/registry/global.go

```diff
@@ -24,6 +24,40 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// Podman is Kubernetes in container using podman driver
+	Podman = "podman"
+	// Docker is Kubernetes in container using docker driver
+	Docker = "docker"
+	// Mock driver
+	Mock = "mock"
+	// None driver
+	None = "none"
+)
+
+// IsKIC checks if the driver is a kubernetes in container
+func IsKIC(name string) bool {
+	return name == Docker || name == Podman
+}
+
+// IsMock checks if the driver is a mock
+func IsMock(name string) bool {
+	return name == Mock
+}
+
+// IsVM checks if the driver is a VM
+func IsVM(name string) bool {
+	if IsKIC(name) || IsMock(name) || BareMetal(name) {
```

Comment:
I know you just copy and pasted the previous function, but because the other functions aren't used, just inline it:

```
if name == Docker || name == Mock || name == None || name == Podman {
  return true
}

Then you can remove IsKic and IsMock.
```